components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -1249,6 +1253,59 @@ public void testChownAllBelongingToUser(boolean isAdmin, boolean permChgrp, bool
         Assert.assertEquals(retrievedProjectDatasetLinkOtherGroup2.getDetails().getOwner().getId().getValue(), recipient.userId);
     }
 
+    /** Test of light amdin without using Sudo.
+     * The workflow deals with the eventuality of putting Rendering Settings on an
+     * image of the user and then transferring the ownership of these settings
+     * to the user.
+     * @throws Exception unexpected
+     */
+    @Test(dataProvider = "narrowed combined privileges cases")
+    public void testRenderingSettingsNoSudo(boolean isAdmin, boolean permChown,
+            boolean permWriteOwned, String groupPermissions) throws Exception {
+        /* creation of rendering settings should be always permitted as long as light admin is in System Group
+         * and has WriteOwned permissions. Exception is Private group, where it will
+         * always fail.*/
+        if (!isAdmin) return;
+        boolean isExpectSuccessCreateRndSettings = isAdmin && permWriteOwned && !(groupPermissions == "rw----") ;
+        boolean isExpectSuccessCreateAndChown = isExpectSuccessCreateRndSettings && permChown;
+        final EventContext normalUser = newUserAndGroup(groupPermissions);
+        /* set up the light admin's permissions for this test */
+        ArrayList <String> permissions = new ArrayList <String>();
+        if (permChown) permissions.add(AdminPrivilegeChown.value);;
+        if (permWriteOwned) permissions.add(AdminPrivilegeWriteOwned.value);
+        final EventContext lightAdmin;
+        lightAdmin = loginNewAdmin(isAdmin, permissions);
```

Comment:
could do this after the image is created to avoid switching back between normal user and admin so much